app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -175,6 +175,21 @@ def alerts_client
       with_provider_connection(&:alerts)
     end
 
+    # server ops
+    def shutdown_middleware_server(ems_ref, params)
+      timeout = params[:timeout] || 0
+      run_generic_operation(:Shutdown, ems_ref, :restart => false, :timeout => timeout)
```

Comment:
jboss cli says:

```bash
./jboss-cli.sh --connect ':read-operation-description(name=shutdown)'
{
    "outcome" => "success",
    "result" => {
        "operation-name" => "shutdown",
        "description" => "Shuts down the server via a call to System.exit(0)",
        "request-properties" => {
            "restart" => {
                "type" => BOOLEAN,
                "description" => "If true, once shutdown the server will be restarted again",
                "expressions-allowed" => false,
                "required" => false,
                "nillable" => true,
                "default" => false
            },
            "timeout" => {
                "type" => INT,
                "description" => "The shutdown timeout in seconds. If this is zero (the default) then the server will shutdown immediately. A value larger than zero means the server will wait up to this many seconds for all active requests to finish. A value smaller than zero means that the server will wait indefinitely for all active requests to finish.",
                "expressions-allowed" => false,
                "required" => false,
                "nillable" => true,
                "default" => 0
            }
        },
        "reply-properties" => {},
        "read-only" => false,
        "runtime-only" => true
    }
}
```

So the default is `false`, I guess the author of the code wanted to be explicit here. It's not my code btw. I only moved the methods that belongs together closer to each other. Sorry for that, the readability of the diff might be bad.